src/sage/groups/perm_gps/permgroup_named.py

```diff
@@ -3435,3 +3438,81 @@ def codegrees(self):
         ret = [self._m * i for i in reversed(range(self._n-1))]
         ret.append((self._n-1)*self._m - self._n)
         return tuple(sorted(ret, reverse=True))
+
+class SmallPermutationGroup(PermutationGroup_generic):
+    def __init__(self, order, gap_id):
+        r"""
```

Comment:
I would move this doc to the class level and have a `TestSuite` doctest in `__init__`.

Reply:
> That’s likely because there are some features missing that are expected. I will take a look tonight.

Thanks!